### PR TITLE
ban non-deterministic precompiled contracts

### PR DESCRIPTION
**Pull request by marie-fourier**
- added a new package "params"

- feat: ban non-deterministic precompiled contracts during userOp validation
https://github.com/etherspot/skandha/blob/141f861195e58a151f78b262ac8f1b37e5682634/packages/params/src/bannedContracts.ts#L1-L22
https://github.com/etherspot/skandha/blob/141f861195e58a151f78b262ac8f1b37e5682634/packages/executor/src/services/UserOpValidation.ts#L456-L474

- env vars for entrypoints and supported networks

### CODE REVIEW

File: packages/params/src/bannedContracts.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+} = {
+  // https://doc.aurora.dev/evm/precompiles/#random-seed
+  aurora: ["0xc104f4840573bed437190daf5d2898c2bdf928ac"], // randomseed
+  auroraTest: ["0xc104f4840573bed437190daf5d2898c2bdf928ac"], // randomseed
+
+  // https://docs.moonbeam.network/builders/pallets-precompiles/precompiles/randomness/
+  moonbeam: ["0x0000000000000000000000000000000000000809"], // randomness
+  moonbase: ["0x0000000000000000000000000000000000000809"], // randomness
+
+  arbitrum: [
```
Comment (ch4r10t33r) [inline packages/params/src/bannedContracts.ts:14]: Wont these be applicable for Optimism as well?

File: packages/types/src/networks/networks.ts [hunk 2]
```diff
@@ -1,33 +1,100 @@
...
+  | "fuse"
+  | "fuseSparknet"
+  | "arbitrumNova"
   | "arbitrumNitro"
+  | "neon"
+  | "neonDevnet"
+  | "optimismGoerli"
+  | "dev"
   | "base"
   | "etherspot";
```
Comment (ch4r10t33r) [inline packages/types/src/networks/networks.ts:32]: We can remove etherspot, sokol, goerli.

**Review by ch4r10t33r — CHANGES_REQUESTED** [review]

File: packages/params/src/bannedContracts.ts (hunk 1 above)
Comment (ch4r10t33r) [reply packages/params/src/bannedContracts.ts:14]: Here are the optimism predeploys: https://github.com/ethereum-optimism/optimism/blob/develop/specs/predeploys.md

Comment (marie-fourier) [reply packages/params/src/bannedContracts.ts:15]: added

File: packages/types/src/networks/networks.ts (hunk 2 above)
Comment (marie-fourier) [reply packages/types/src/networks/networks.ts]: done

**Review by ch4r10t33r — APPROVED** [review]

**Pull request merged**